docs/copied-from-beats/shared-config-ingest.asciidoc

```diff
@@ -0,0 +1,71 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/shared-config-ingest.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+[[configuring-ingest-node]]
+== Parse data by using ingest node
```

Comment:
did we test this with apm server?

Reply:
Yes, in #1032

Reply:
nice example you got there! totally missed it